suite-native/module-send/src/components/RecipientsSummary.tsx

```diff
@@ -78,20 +84,24 @@ export const RecipientsSummary = ({ accountKey, selectedFeeLevel }: FeesRecipien
                                 {output.address}
                             </Text>
                         </VStack>
-                        <VStack flex={0.4} alignItems="flex-end" spacing="sp4">
-                            <CryptoToFiatAmountFormatter
+                        <VStack flex={0.6} alignItems="flex-end" spacing="sp4">
+                            <CoinToFiatAmountFormatter
                                 variant="hint"
                                 color="textDefault"
                                 value={output.amount}
-                                network={networkSymbol}
                                 isBalance={isBalance}
+                                accountKey={accountKey}
+                                tokenContract={tokenContract}
                             />
-                            <CryptoAmountFormatter
+                            <CoinAmountFormatter
                                 variant="hint"
                                 color="textSubdued"
                                 value={output.amount}
-                                network={networkSymbol}
                                 isBalance={isBalance}
+                                accountKey={accountKey}
+                                tokenContract={tokenContract}
+                                adjustsFontSizeToFit
+                                numberOfLines={1}
```

Comment:
Is this something we'll ever want to set to value other than 1? 🤔 Would consider removing from the component's interface.

Reply:
These formatters are only reusing the [Text component](https://github.com/trezor/trezor-suite/blob/1996ba9d97bb1d604a0673680ebec0f6fc5cd106/suite-native/atoms/src/Text.tsx) API. If we would like to always set this prop equal to 1 if the `adjustFontSizeToFit` is true, we should refactor every usage of it in the repo. I'm not saying I'm against it, but not sure if it is worth it. Feel free to create an issue for that, and we can discuss it during the next standup meeting.

Reply:
This components are wrapped in our own, if you set it as default in `CoinAmountFormatter` and `CoinToFiatAmountFormatter` it shoudn't be issue and it shoudn't require any fefactorings/

Reply:
Some refactoring might make sense but as a follow-up. I was asking in the context of this change.